### ISSUE TITLE
Jfrog-cli to support buildah (alternative to docker) to push docker images to artifactory

### ISSUE THREAD

**Issue by vigneshsweekaran** [jfrog/jfrog-cli#1962]
We are using only **buildah** in our environment to build and push the docker images. Now we are planning to use jfrog-cli to publish and scan the docker images. But jfrog-cli tightly coupled with docker. So we are looking for jfrog-cli to support buildah

jfrog-cli should have the additional option to use buildah to push the docker images to artifactory. And even the build-info should be published along with that

Expected :
1. With buildah support jfrog-cli should be able to push images to artifactory along with build-info
2. With buildah support jfrog-cli should be able to scan the docker images in artifactory

**Comment by michaelfranzl** [comment]
This already works. Images built with `buildah` can be pushed with `jf rt pp` because podman and buildah access the same image storage.

**Comment by cospeedster** [comment]
@michaelfranzl Thats true but you still rely on `podman` cli and cannot use only `buildah` cli in your build environment.

**Comment by netflash** [comment]
I would like to ressurect this - any chance the buildah support is on the radar?
